src/devhub/page/admin.jsx

```diff
@@ -0,0 +1,134 @@
+const { useQuery, hasModerator } = VM.require(
+  "${REPL_DEVHUB}/widget/core.adapter.devhub-contract"
+);
+
+const { Tile } =
+  VM.require("${REPL_DEVHUB}/widget/devhub.components.molecule.Tile") ||
+  (() => <></>);
+
+const { href } = VM.require("${REPL_DEVHUB}/widget/core.lib.url");
+
+if (!useQuery || !hasModerator || !href) {
+  return <p>Loading modules...</p>;
+}
+
+const AdministrationSettings = {
+  communities: {
+    maxFeatured: 4,
+  },
+};
+
+const CommunityFeaturingSchema = {
+  handle: {
+    label: "Community handle",
+
+    hints: {
+      disabled: `You can only add ${AdministrationSettings.communities.maxFeatured} communities at a time`,
+    },
+
+    inputProps: { min: 3, max: 40, required: true },
+  },
+};
+
+const featuredCommunities = useQuery("featured_communities");
```

Comment:
AC says that the changes sould be applied instantaneously. Plus, state actualization is requirted to preselve consistency, otherwise you may end up rewriting changes made by someone else before you